profiles/views.py

```diff
@@ -57,21 +62,40 @@ def test_func(self) -> bool:
 
     def get(self, request: HttpRequest, profile: str) -> HttpResponse:
         user = get_object_or_404(ForumUser, username=profile)
-        form = EditProfileForm(instance=user)
-        return render(request, self.template_name, {"user": user, "form": form})
+        edit_profile_form = EditProfileForm(instance=user)
+        user_profile = get_object_or_404(Profile, user=self.request.user)
+        edit_links_form = EditProfileLinksForm(instance=user_profile)
+        return render(
+            request,
+            self.template_name,
+            {
+                "user": user,
+                "edit_profile_form": edit_profile_form,
+                "edit_links_form": edit_links_form,
+            },
+        )
 
     def post(self, request: HttpRequest, profile: str) -> HttpResponse:
         user = get_object_or_404(ForumUser, username=profile)
-        form = EditProfileForm(request.POST, instance=user)
+        edit_profile_form = EditProfileForm(request.POST, instance=user)
+        user_profile = get_object_or_404(Profile, user=self.request.user)
+        edit_links_form = EditProfileLinksForm(request.POST, instance=user_profile)
 
-        if form.is_valid():
-            user = form.save(commit=False)
+        if edit_profile_form.is_valid() and edit_links_form.is_valid():
+            user = edit_profile_form.save(commit=False)
+            user_profile = edit_links_form.save(commit=False)
+            user_profile.user = self.request.user
+            user_profile.save()
```

Comment:
some incorrect namings there
```suggestion
            user_profile = edit_profile_form.save(commit=False)
            user_links = edit_links_form.save(commit=False)
            user_profile.user = self.request.user
            user_profile.save()
            user_links.save()
```